apis/cluster/v1beta1/membercluster_types.go

```diff
@@ -69,6 +74,21 @@ type MemberClusterStatus struct {
 	AgentStatus []AgentStatus `json:"agentStatus,omitempty"`
 }
 
+// Taint attached to MemberCluster has the "effect" on
+// any ClusterResourcePlacement that does not tolerate the Taint.
+type Taint struct {
+	// The taint key to be applied to a MemberCluster.
+	// +required
+	Key string `json:"key"`
+	// The taint value corresponding to the taint key.
+	// +optional
+	Value string `json:"value,omitempty"`
+	// The effect of the taint on ClusterResourcePlacements that do not tolerate the taint.
+	// Valid effect is NoSchedule.
+	// +required
+	Effect corev1.TaintEffect `json:"effect"`
```

Comment:
can you add kubebuilder validation marker?

eg, 
//+kubebuilder:validation:Enum=PickAll;PickN;PickFixed